includes/admin/controllers/class-admin-fees.php

```diff
@@ -88,16 +119,18 @@ private function insert_or_update_fee( $mode ) {
 			if ( ! isset( $posted_values['recurring'] ) ) {
                 $posted_values['recurring'] = 0;
 			}
+			$images = (int) $posted_values['images'];
         } else {
             $posted_values = array();
+			$images = false;
         }
 
-		$images_changed = false;
 		if ( 'insert' === $mode ) {
             $fee = new WPBDP__Fee_Plan( $posted_values );
+			$images_changed = false;
         } else {
-			$fee            = $this->get_or_die();
-			$images_changed = (int) $fee->images !== (int) $posted_values['images'];
+			$fee = $this->get_or_die();
+			$images_changed = $images !== false && (int) $fee->images !== $images;
```

Comment:
@theRealRizeo The changes lines from 122-133 don't look related to this PR. Can you please move this out to a new branch? I'm not sure what this is for.

Reply:
Hey @stephywells It' a change you committed https://github.com/Strategy11/business-directory-plugin/commit/58a9e591102cfef35c23160f554e7495c7d2fdf3 I will revert it

Reply:
Last commit reverted to fix for undefined variable